package-lock.json

```diff
@@ -7661,7 +7661,7 @@
     },
     "external-editor": {
       "version": "2.2.0",
-      "resolved": "https://registry.npmjs.org/external-editor/-/external-editor-2.2.0.tgz",
+      "resolved": "http://registry.npmjs.org/external-editor/-/external-editor-2.2.0.tgz",
```

Comment:
This is a setting that has bitten us previously.  These should all be `https` not `http`

Reply:
yeah..